snap/hooks/post-refresh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -e
+
+# force remove kong-admin-jwt due to the fact that it's
+# written by security-secretstore-setup with read-only
+# perm for the owner. This is hook is called in the
+# context of the new snap revision, and prior to
+# services being started.
```

Comment:
> This ~is~ hook is called in the

It would be good to mention that this is/was an issue up with v2.0.0:

> force remove kong-admin-jwt due to the fact that it's
written by **v2.0.0 of** security-secretstore-setup with read-only
perm for the owner. This hook is called in the